configuration/exodus-pipeline.yaml

```diff
@@ -126,10 +117,13 @@ Resources:
                 Configuration:
                   S3Bucket: exodus-pipeline-artifacts
                   S3ObjectKey:
-                    !FindInMap [SourceMap, !Ref env, objectKey]
-                  PollForSourceChanges: false
+                    !If
+                      - NotProd
+                      - build-dev
+                      - build-stage
+                  PollForSourceChanges: true
```

Comment:
Do you know whether this PollForSourceChanges is necessary?  If so, what's the poll interval?  I thought this is typically event-driven and not relying on polling.

Reply:
I thought so as well but in practice didn't see the event trigger working. I'll give it another try today.

Reply:
So, event-driven change detection isn't automatic in our case and will require the creation of a CloudWatch event in the pipeline template.

I'd like to add this instead of relying on polling (the interval for which isn't made clear anywhere) but will submit a follow-up PR with that change.